activejob/lib/active_job/queue_adapters.rb

```diff
@@ -116,6 +116,14 @@ module QueueAdapters
     private_constant :ADAPTER
 
     class << self
+      # Returns adapter for specified name. For example, this enables to perform
+      # job with specific adapter:
+      #
+      #   Adapter = ActiveJob::QueueAdapters.lookup(:sidekiq)
+      #   # => ActiveJob::QueueAdapters::SidekiqAdapter
+      #
+      #   Adapter.enqueue DeliverMail.new
```

Comment:
I don't think we need to document the adapter behavior. This method is only about lookup.